lib/_stream_legacy.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
```

Comment:
That makes sense, especially since everything in the file is exported. Nevermind! :)